app/controllers/collaborators_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CollaboratorsController < ApplicationController
+  def index
+    @collaborators = current_user.collaborators
+  end
+
+  def resend_invitation
```

Comment:
Move this action to its own controller (e.g `InvititationResendsController#create`, we only want to use the RESTful actions.